### PR TITLE
Add python psutil to developer packages

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Add `psutil` to developer requirements.

**To test:**

Review the changes.

*No associated issue*

Does this update require release notes?
- [ ] Yes
- [X] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: buildconfig/dev-packages/deb/mantid-developer/ns-control [hunk 1]
```diff
@@ -61,6 +61,7 @@ Depends: git,
   python-h5py,
   python-yaml,
   python-mock,
+  python-psutil,
```
Comment (rosswhitfield) [inline buildconfig/dev-packages/deb/mantid-developer/ns-control]: And `python3-psutil`?

Comment (martyngigg) [reply buildconfig/dev-packages/deb/mantid-developer/ns-control]: Thanks. I'd forgotten we'd added the python3 dependencies now.

**Review by AndreiSavici — APPROVED** [review]

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec [hunk 2]
```diff
@@ -103,6 +104,7 @@ Requires: python3-matplotlib
 %{?fedora:Requires: python3-qtconsole}
 Requires: python3-PyYAML
 Requires: python3-mock
+Requires: python3-psutil
```
Comment (peterfpeterson) [inline buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: This needs to be wrapped in a fedora check like `python3-qtconsole`

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
I think the package suggested by the top level `CMakeLists.txt` has the wrong name for the

File: buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec (hunk 2 above)
Comment (martyngigg) [reply buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec]: Just out of curiosity how come some of the other `python3` packages are not wrapped?

**Review by peterfpeterson — APPROVED** [review]
Good for RHEL7 now

**Pull request merged**